### PR TITLE
deploy on secret changes

### PR DESCRIPTION
**Pull request by vihangvk**
#21  I have added annotation in deployment as per https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments #

### CODE REVIEW

**Comment by JorritSalverda** [comment]
Can you assign this to yourself and update the version and release branch in the estafette manifest to that of the milestone 1.2.3? I've already created the milestone.

File: helm/estafette-cloudflare-dns/templates/deployment.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ spec:
       annotations:
         prometheus.io/scrape: "true"
         prometheus.io/port: "9101"
+        checksum/secrets: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
```
Comment (JorritSalverda) [inline helm/estafette-cloudflare-dns/templates/deployment.yaml:30]: Does this get evaluated after the values are filled into the secret?

Comment (vihangvk) [reply helm/estafette-cloudflare-dns/templates/deployment.yaml:30]: yes, tested with different values using Helm template.

**Review by lruslan — APPROVED** [review]

File: helm/estafette-cloudflare-dns/templates/deployment.yaml (hunk 1 above)
Comment (JorritSalverda) [reply helm/estafette-cloudflare-dns/templates/deployment.yaml:30]: Excellent, let's apply this to the other official Estafette Helm charts making use of secrets and configmaps as well (unless the application can deal with config changes, but none of the current 9 controllers do).

**Review by JorritSalverda — APPROVED** [review]

**Pull request merged**